src/analyze/analyze-plot.c

```diff
@@ -464,6 +492,64 @@ static int produce_plot_as_text(UnitTimes *times, const BootTimes *boot) {
         return show_table(table, "Units");
 }
 
+static int help(void) {
+        printf("%s plot [OPTIONS]\n\n"
+               "Output SVG graphic showing service initialization\n"
+               "  -h --help                                Show this help\n"
+               "  -f --hide-firmware                       Do not plot firmware events\n"
+               "  -l --hide-loader                         Do not plot bootloader events\n"
+               "  -k --hide-kernel                         Do not plot kernel events\n"
+               "  -i --hide-initrd                         Do not plot initrd events\n"
+               "  -i --hide-reboot                         Do not plot soft-reboot events\n",
+               program_invocation_short_name);
+
+        return 0;
+}
+
+static int parse_argv(int argc, char *argv[]) {
```

Comment:
Could you merge the parser to the global one? We usually do not do like that (except for `udevadm`).

Reply:
As far as I can tell, `udevadm` is the only other place where verbs use `getopt_long()` to parse their arguments (which I believe is the right thing to do in this case*) and like `systemd-analyze` one of few which implement their verbs in separate files. These options don't make sense in a context of any other other `systemd-analyze` verb. This makes me assume the option parsing as well as `arg_hide_*` variables should be implemented in `systemd-analyze.c` 

* However I am not sure whether separate options should be used or rather `-H [PART], --hide=[PART]`.

Reply:
Done.